### DISCUSSION TITLE
Strange internal perimeters and gap fill in top layer under text

### DISCUSSION THREAD

**Discussion by OverWorldD** [supermerill/SuperSlicer#1896] [Q&A] [answered]
So I'm printing some buttons with raised text. The top layer under the buttons is ironed. Now, strangely enough there are small areas of internal perimeters and a gap fill in the top layer. They are not present in the layer underneath the top layer, nor are they connected to anything in the text layer above. 

Do we have any setting that might cause/fix this?

![Untitled](https://user-images.githubusercontent.com/12225528/142024252-5adc77ea-e545-4ba7-be5a-766be0b3eebd.png)
m

**Comment by supermerill** [comment, marked as answer]
it's because of the `min_width_top_surface` from the "only-one-perimter-on-top".
Set it to 0 to disable the algo that tries to limit it to not-small surfaces.
Or just uncheck `only_one_periemter_top`.

It's an unavoidable side-effect, the letters are enlarged and they merge and then they don't totally collapse when shrunk.

**Reply by OverWorldD** [reply]
Late reply but thank you!
